src/components/crossDevice/CrossDeviceLink/index.js

```diff
@@ -177,22 +177,22 @@ class CrossDeviceLinkUI extends Component {
       alert(`No SMS will be sent, please copy this link ${window.location.origin}`)
     }
 
-    const { language, sms, token } = this.props
+    const { language, sms, token, urls } = this.props
     const options = {
       payload: JSON.stringify({to: sms.number, id: this.linkId, language}),
-      endpoint: `${process.env.SMS_DELIVERY_URL}/v1/cross_device_sms`,
+      endpoint: `${urls.telephony_v1_url}/v1/cross_device_sms`,
```

Comment:
hm, how come it's `telephony_v1_url` yet `/v1/` it's still added as a path?

Reply:
That was my thought but the key returned by the backend is `telephony_v1_url` and does not include the version